### DISCUSSION TITLE
Log-File available for your application / tool?

### DISCUSSION THREAD

**Discussion by beinando** [kangyu-california/PersistentWindows#147] [Ideas]
Hey man, first thanks for your tool, i think there's a lot of people who can have a benefit from it. I'd just like to know if it's  possible for your tool to write a log-file in case it does an action, like restoring the previously saved monitor settings. We're using this for a sine-projection which sometimes switches monitors and we'd like to know how often this occurs and if the tool actually does react properly to the change of display settings within windows. 
Thanks already in advance!

**Comment by kangyu-california** [comment]
PW log can be found in  Event Viewer, search for Application event id 9990 and 9999,
![image](https://user-images.githubusercontent.com/59128756/154363471-b2964c39-340f-4eaf-b9fc-b4f725a79ac8.png)

Events can be filtered and exported in txt/xml format
